Extension/src/LanguageServer/client.ts

```diff
@@ -1338,8 +1338,8 @@ export class DefaultClient implements Client {
             defaultSystemIncludePath: settings.defaultSystemIncludePath,
             cppFilesExclude: settings.filesExclude,
             clangFormatPath: util.resolveVariables(settings.clangFormatPath, this.AdditionalEnvironment),
-            clangFormatStyle: util.resolveVariables(settings.clangFormatStyle, this.AdditionalEnvironment),
-            clangFormatFallbackStyle: settings.clangFormatFallbackStyle,
+            clangFormatStyle: settings.clangFormatStyle ? util.resolveVariables(settings.clangFormatStyle, this.AdditionalEnvironment) : undefined,            
+            clangFormatFallbackStyle: settings.clangFormatFallbackStyle ? settings.clangFormatFallbackStyle : undefined,
```

Comment:
`clangFormatFallbackStyle` shouldn't change though, right? Please change it back.

Reply:
Yeah, this change appears to be unnecessary.

Reply:
Based on what we discussed, if we don't return undefined here, we will pass through an empty string and it won't fallback to the default defined in the native code, it should be the same bug as clangFormatStyle.

Reply:
Please add this to your notes for the follow up PR. We are avoiding changing behavior in this PR. Leaving this line alone won't regress anything, will it?

Reply:
@browntarik settings.clangFormatFallbackStyle is implemented as `changeBlankStringToUndefined` so the changed code is unnecessary.

Reply:
@sean-mcmanus it looks like you're right. I made a bad assumption, I should have tested it first.